app/views/layouts/_top_bar.html.erb

```diff
@@ -58,7 +58,8 @@
         </a>
 
         <a href="<%= sign_up_path(state: "new-user") %>" class="crayons-btn mr-2 whitespace-nowrap" data-no-instant>
-          Create account
+          <span class="button-text">Create account</span>
+          <span class="button-text-wide-logo">Join</span>
```

Comment:
We use utility-first classes (learn more [here](https://tailwindcss.com/)*) in our codebase so you should be able to achieve the very same results with this code below instead of writing that custom CSS.

Let me know if it makes sense and if it works.

```suggestion
          <span class="hidden s:inline">Create account</span>
          <span class="s:hidden">Join</span>
```

* - we don't really use Tailwind but we adapted their approach.